tests/expectations/test_expectation_diagnostics.py

```diff
@@ -142,11 +134,19 @@ def test__convert_checks_into_output_message():
             ),
         ],
     )
+)
+
+
+def test_ExpectationDiagnosticReport():
 
+    print(edr)
```

Comment:
?

Reply:
git did a funky diff on that... [see this chunk of code](https://github.com/great-expectations/great_expectations/blob/96423d6b706725f0411450f81cbd58d743e3ad2b/tests/expectations/test_expectation_diagnostics.py#L61-L159)

In `test__convert_checks_into_output_message`, an `ExpectationDiagnosticMaturityMessages` object is defined, then it is manually passed into `edr._convert_checks_into_output_message(`. I took that object and passed it in as the new `maturity_checklist` parameter to the instance of `ExpectationDiagnostics` being created (`edr`).

Then in `test__convert_checks_into_output_message` I pass `edr.maturity_checklist` instead of `checks`.

The `print(edr)` is just what was always in `test_ExpectationDiagnosticReport()`

Reply:
Got it. Let's just delete it, since it's a no-op test.